ui/partials/main/tcjobactions.html

```diff
@@ -19,9 +19,10 @@ <h4 class="modal-title">Custom Taskcluster Job Actions</h4>
   {{input.selectedAction.jsonPayload}}
 </div>
 <div class="modal-footer">
-  <button class="btn btn-primary" ng-click="triggerAction()" ng-attr-title="{{user.loggedin ? 'Trigger this action' : 'Not logged in'}}" ng-disabled="triggering || !user.loggedin">
+  <button ng-if="user.loggedin" class="btn btn-primary" ng-click="triggerAction()" ng-attr-title="{{user.loggedin ? 'Trigger this action' : 'Not logged in'}}" ng-disabled="triggering">
     <span class="glyphicon glyphicon-check" aria-hidden="true"></span>
     <span ng-if="triggering">Triggering</span>
     <span ng-if="!triggering">Trigger</span>
   </button>
+  <p ng-if="!user.loggedin" class="help-block">Custom actions require login</p>
```

Comment:
I would consider using a bootstrap alert like this one here (probably in the modal body) since it's a little more obvious:

https://github.com/mozilla/treeherder/blob/5de04f59403de5b70a57e0b608d0b1f6079b81a6/ui/partials/perf/alertsctrl.html#L2

Up to you though.